app.py

```diff
@@ -1,82 +1,97 @@
 
+
 @app.route('/authorize')
 def authorize():
     code = request.args.get('code')
     access_token = get_access_token(code)
     session['access_token'] = access_token
     return redirect('/playlist')
 
+
 @app.route('/playlist')
 def playlist():
     if not 'access_token' in session:
         return redirect('/')
     access_token = session['access_token']
-    recomendation_music = get_Recommendations(access_token)
+    recommendation_music = get_Recommendations(access_token)
     for next_music in range(50):
-        get_music_queue(access_token,recomendation_music[next_music]['id'])
-    return 'done'
+        get_music_queue(access_token, recommendation_music[next_music]['id'])
+    return 'Done!!!!'
+
 
 def get_Recommendations(token):
     try:
-        actual_song = get_actual_play_music(token)
-        seed_artists = '?seed_artists=' +actual_song['artists'][0]['id']
-        seed_tracks = '&seed_tracks=' + actual_song['id']
-        url= "https://api.spotify.com/v1/recommendations" +seed_artists + seed_tracks +'&limit=50'
-        headers = {'Authorization': 'Bearer '+token , 'content_type': 'application/json'}
-        response = requests.get(url , headers=headers)
+        current_song = get_current_play_music(token)
+        seed_artists = '?seed_artists=' + current_song['artists'][0]['id']
+        seed_tracks = '&seed_tracks=' + current_song['id']
+        url = "https://api.spotify.com/v1/recommendations" + \
+            seed_artists + seed_tracks + '&limit=50'
+        headers = {'Authorization': 'Bearer ' +
+                   token, 'content_type': 'application/json'}
+        response = requests.get(url, headers=headers)
         song = response.json()
         return song['tracks']
-    except requests.exceptions.HTTPError as e:
-        print (e)
+    except requests.exceptions.HTTPError as err:
+        print(err)
+
 
-def get_actual_play_music(token):
+def get_current_play_music(token):
     try:
         url = "https://api.spotify.com/v1/me/player/currently-playing"
-        headers = {'Authorization': 'Bearer ' +token , 'content_type': 'application/json'}
-        respone =requests.get(url, headers=headers)
+        headers = {'Authorization': 'Bearer ' +
+                   token, 'content_type': 'application/json'}
+        respone = requests.get(url, headers=headers)
         music = respone.json()
         return music['item']
-    except requests.exceptions.HTTPError as e:
-        print (e)
+    except requests.exceptions.HTTPError as err:
+        print(err)
 
 
-def get_music_queue(token,song):
+def get_music_queue(token, song):
     try:
-        music = '?uri=spotify:track:'+song 
-        url = "https://api.spotify.com/v1/me/player/queue"+music
-        headers = {'Authorization': 'Bearer ' +token , 'content_type': 'application/json'}
+        music = '?uri=spotify:track:' + song
+        url = "https://api.spotify.com/v1/me/player/queue" + music
+        headers = {'Authorization': 'Bearer ' +

```

Comment:
This is copied like 3th time. Should be moved into function.  (headers management)